src/test/java/IBHW18.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class IBHW18 extends BaseTest{
+
+    @Test
+    public void playASongTest() {
+        login("ionut.burtoiu@testpro.io", "te$t$tudent");
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/216551610-60799ba8-3465-4011-a1ba-7535e0623cf7.png)